pom.xml

```diff
@@ -243,7 +251,8 @@
                                 <phase>generate-sources</phase>
                                 <configuration>
                                     <sources>
-                                        <source>${project.basedir}/src/main/spark30+all/scala</source>
+                                        <source>${project.basedir}/src/main/spark301+311-all/scala</source>
```

Comment:
would it be easier for others to understand just to use words?  301until311 (exclusive)  301thru312 (inclusive)

Reply:
note not a blocker, just throwing the idea out there, we can always rename later.

Reply:
I like the idea, it looks more readable to me except I lean more towards `301to312` instead of informal `thru`  for brevity, and Scala range syntax analogy. Depending if there will be more blocking comments I'll incorporate the suggestion.